pymeasure/instruments/exfo/ctp10.py

```diff
@@ -0,0 +1,1031 @@
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+
+from pymeasure.instruments import Instrument, SCPIMixin
+from pymeasure.instruments.channel import Channel
+from pymeasure.instruments.validators import strict_range
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class RLASerChannel(Channel):
+    """Represents a RLASer (Reference Laser) channel on the EXFO CTP10.
+
+    Valid channel numbers: 1-10.
+
+    In laser sharing mode, this query is not available on Distributed CTP10s.
+    """
+
+    idn = Channel.measurement(
+        ":CTP:RLASer{ch}:IDN?",
+        """Get the identification of the laser (str).
+
+        Returns a comma-separated string with format: manufacturer,model,serial,firmware
+
+        Example response: "EXFO,T100S-HP,0,6.06"
+
+        Response components:
+        - manufacturer: Manufacturer of the laser
+        - model: Instrument model
+        - serial: Instrument serial number
+        - firmware: Instrument firmware version
+        """,
+    )
+
+    power_dbm = Channel.control(
+        ":CTP:RLASer{ch}:POWer?",
+        ":CTP:RLASer{ch}:POWer %gDBM",
+        """Control the laser output power in dBm (float).
+
+        Sets the laser output power value in static control, or returns the power value
+        set for the given laser in static control, or the actual laser power after an
+        acquisition.
+
+        Possible values depend on the laser specifications.
+
+        On T500S, the maximum power is limited to 13 dBm. To avoid permanent damage
+        to the CTP10 module detectors, do not apply a higher output power value than
+        the maximum safe power specified for the detector to which the laser is connected
+        (refer to Optical Measurement Specifications).
+
+        The default unit is dBm.
+        """,
+    )
+
+    power_mw = Channel.control(
+        ":CTP:RLASer{ch}:POWer?",
```

Comment:
The getter is the same as above, right?

So what value will you actually get? In dBm or in mW?